src/clj/qbits/jet/client/http.clj

```diff
@@ -94,22 +94,16 @@
       ([result chunk]
        (reduction-function result (decode-body chunk as))))))
 
-(defrecord Response [status headers body error-chan])
-
-(defn ^:no-doc result->response
-  [^Result result body-ch]
-  (let [response (.getResponse result)
-        error-ch (async/promise-chan)]
-    (if (.isSucceeded result)
-      (async/close! error-ch)
-      (async/put! error-ch {:error (.getFailure result)}))
-    (Response. (.getStatus response)
-               (reduce (fn [m ^HttpField h]
-                         (assoc m (string/lower-case  (.getName h)) (.getValue h)))
-                       {}
-                       ^HttpFields (.getHeaders response))
-               body-ch
-               error-ch)))
+(defrecord RResponse [request status headers body error-chan])
```

Comment:
I think it's better to use the fully-qualified import name than change the name of the record here.